tap_salesforce/salesforce/__init__.py

```diff
@@ -381,13 +384,20 @@ def _get_selected_properties(self, catalog_entry):
                                             self.select_fields_by_default)]
 
 
-    def get_start_date(self, state, catalog_entry):
+    def get_start_date(self, state, catalog_entry, without_lookback=True):
```

Comment:
Add code comment and explain the effect of the without_lookback variable.

Reply:
Added function comment